.github/workflows/frontend.yml

```
@@ -59,6 +59,37 @@ jobs:
           start-time: ${{ env.START_TIME }}
           max-seconds: '600'
 
+  frontend-locales:
+    name: Frontend Locale Parity + Coverage
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Start Timer
+        run: echo "START_TIME=$(date +%s)" >> $GITHUB_ENV
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:34: Double quote to prevent globbing and word splitting

(shellcheck)